gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorage.java

```diff
@@ -30,24 +30,22 @@
  * https://developers.google.com/storage/docs/concepts-techniques
  */
 public interface GoogleCloudStorage {
-  // Pseudo path delimiter.
-  //
-  // GCS does not implement full concept of file system paths but it does expose
-  // some notion of a delimiter that can be used with Storage.Objects.List to
-  // control which items are listed.
-  public static final String PATH_DELIMITER = "/";
 
   /**
-   * Value indicating all objects should be returned from GCS, no limit.
+   * Pseudo path delimiter.
+   *
+   * <p>GCS does not implement full concept of file system paths but it does expose some notion of a
+   * delimiter that can be used with ${code Storage.Objects.List} to control which items are listed.
    */
-  public static final long MAX_RESULTS_UNLIMITED = -1;
+  String PATH_DELIMITER = "/";
```

Comment:
Why these constant modifiers got removed?

Reply:
They are applied by default in Java interfaces - Intellij linter shows them as redundant warning.